sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/TestHelper.cs

```diff
@@ -168,6 +128,22 @@ public static async Task<HttpResponseMessage> EventResponseBaseTest(HttpMethods
             }));
         }
 
+        internal static AuthenticationEventResponseHandler GetAuthenticationEventResponseHandler(HttpRequestMessage mockedRequest)
+        {
+            AuthenticationEventResponseHandler eventsResponseHandler = null;
+#if NET461
```

Comment:
```suggestion
#if NETFRAMEWORK
```

Reply:
Generally, the distinction we try to make is between "desktop framework' and "modern .NET runtimes"

Reply:
updated. resolving comment